sigmastate/src/main/scala/org/ergoplatform/ErgoAddress.scala

```diff
@@ -108,7 +108,7 @@ class P2PKAddress(val pubkey: ProveDlog,
   override def networkPrefix: NetworkPrefix = encoder.networkPrefix
 
   override def equals(obj: Any): Boolean = obj match {
-    case p2pk: P2PKAddress => util.Arrays.equals(pubkeyBytes, p2pk.pubkeyBytes)
+    case p2pk: P2PKAddress => java.util.Arrays.equals(pubkeyBytes, p2pk.pubkeyBytes)
```

Comment:
import java.util now not used but not removed (I recommend to start using automated checkers for unused imports, finally, like done in the node repo)

Reply:
fixed. Also configured SBT to warn about unused imports. Unfortunately I cannot make it errors, because there a lot of warnings of different kind, but getting rid of them better be done in separate PR.